azure-devops/projects/cgn-onboarding-portal-projects/secrets.tf

```
@@ -12,6 +12,10 @@ variable "secrets" {
     "io-azure-devops-github-USERNAME",
     "pagopa-npm-bot-TOKEN",
     "TTDIO-SPN-TENANTID",
+    "TTDIO-UAT-GCNPORTAL-SUBSCRIPTION-ID",
+    "TTDIO-PROD-GCNPORTAL-SUBSCRIPTION-ID",
```

```suggestion
    "PAGOPAIT-TENANTID",
    "PAGOPAIT-UAT-GCNPORTAL-SUBSCRIPTION-ID",
    "PAGOPAIT-PROD-GCNPORTAL-SUBSCRIPTION-ID",
```
I think that npm connection is not needed
*-CGNPORTAL are on another tenant, pagopa.it